testsuite/tests/typing-short-paths/cycles.ml

```diff
@@ -0,0 +1,116 @@
+(* TEST
+ flags = " -short-paths ";
+ expect;
```

Comment:
In order to avoid the long timeout on failure, you can limit the stack size by adding
```
ocamlrunparam +="l=100000";
```